src/main/resources/flattening/config/cmap/env.conf

```diff
@@ -37,6 +37,6 @@ join = [
     name = "MCO_CE"
     join_keys = ["ETA_NUM", "SEQ_NUM"]
     main_table_name = "MCO_CSTC"
-    tables_to_join = ["MCO_FMSTC", "MCO_FASTC", "MCO_FBSTC"]
+    tables_to_join = ["MCO_FMSTC", "MCO_FASTC"]
```

Comment:
Add  "MCO_FBSTC" to the tables.

Reply:
do we need MCO_FBSTC in the cmap  ? this one is only used in bulk for extracting ATU event